WorkforceDataApi/src/WorkforceDataApi.DevUtils/Csv/EstablishmentsCsvDownloader.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Net.Http;
+
+namespace WorkforceDataApi.DevUtils.Csv;
+
+public class EstablishmentsCsvDownloader : IEstablishmentsCsvDownloader
+{
+    private const string BaseDownloadUri = "https://ea-edubase-api-prod.azurewebsites.net/edubase/downloads/public/";
+    private readonly IHttpClientFactory _httpClientFactory;
+
+    public EstablishmentsCsvDownloader(IHttpClientFactory httpClientFactory)
```

Comment:
```suggestion
    public EstablishmentsCsvDownloader(HttpClient httpClient)
```

e.g. https://github.com/DFE-Digital/get-an-identity/blob/895757e7bd3d9f8d2d3e484db8df8efce269c4cb/dotnet-authserver/src/TeacherIdentity.AuthServer/Services/DqtApi/ServiceCollectionExtensions.cs#L22

Reply:
Done